containers/CSharp/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,74 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet
+{
+    "name": "C# (.NET)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": { 
+            // Update 'VARIANT' to pick a .NET Core version: 2.1, 3.1, 5.0
+            "VARIANT": "3.1",
+            // Options
+            "NODE_VERSION": "none",
+            "INSTALL_AZURE_CLI": "true"
+        }
+    },
+    "runArgs": [
+        "--network",
+        "host"
+    ],
+
+    "mounts": [
+        // Keep command history 
+        "source=ostf-bashhistory,target=/commandhistory,type=volume",
+        // Use hosts docker socket
+        "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
+    ],
+
+    // Set *default* container specific settings.json values on container create.
+    "settings": {},
+
+    // Add the IDs of extensions you want installed when the container is created.
+    "extensions": [
```

Comment:
>extensions [](http://example.com/codeflow?start=5&length=10)

these extensions are installed on the VSCode instance that is running inside the container, not the dev machine one, right?

Reply:
Yes - I tested this with a bare-bone VSCode installation to verify :-)